### PR TITLE
Fix: jq has been missing in the installer scripts

### PR DESCRIPTION
**Pull request by tunguyen9889**
The installer script which was updated in https://github.com/gravitational/teleport/pull/39033 has 2 issues:
- Missing `jq` package in the `no automatic upgrades` condition.
- Forgot to remove `sudo zypper --non-interactive install ${PACKAGE_LIST}` line when removed the `$PACKAGE_LIST` var.

### CODE REVIEW

**Review by hugoShaka — APPROVED** [review]

**Review by tigrato — APPROVED** [review]

**Comment by tunguyen9889** [comment]
Thanks @hugoShaka and @tigrato for approving this! But I'm not sure if it needs approval from maintainers before merging, as the workflow failed with code:

```
2024/03/19 18:38:10 Workflow check failed: at least two approvals required from [jimbishopp klizhentas r0mant russjones zmb3].
```

**Review by r0mant — APPROVED** [review]

**Comment by hugoShaka** [comment]
Hi @tunguyen9889,

Thanks for your contribution. We are currently unable to run our tests on
external contributions. As such, I'll be helping you validate and
merge your pull request. I've created an internal PR here:

[<link to internal PR>](https://github.com/gravitational/teleport/pull/39594)

Please and I'll get your code merged. You will retain authorship of your commits.

**Pull request closed without merging**
